test/L2/L2Claim.t.sol

```diff
@@ -191,6 +191,23 @@ contract L2ClaimTest is Test {
         assertEq(l2Claim.version(), "1.0.0");
     }
 
+    function test_ClaimRegularAccount_RevertWhenZeroLengthProof() public {
+        uint256 accountIndex = 0;
+        MerkleTreeLeaf memory leaf = getMerkleLeaves().leaves[accountIndex];
+        Signature memory signature = getSignature(accountIndex);
+
+        leaf.proof[0] = bytes32AddOne(leaf.proof[0]);
```

Comment:
this line is redundant for this scenario

Reply:
Nice catch!
Done.